internal/testutil/test_envar_keys.go

```diff
@@ -0,0 +1,175 @@
+package testutil
+
+//########################################
+//###### PORTS / CROSS CONNECT / VC
+//########################################
+
+// PORTS
+const PF_PORT_MEDIA_KEY = "PF_PORT_MEDIA"
+const PF_PORT_POP1_KEY = "PF_PORT_POP1"
```

Comment:
I assume that Packetfabric pop/zone should be selected by the test itself and not provided externally. Port/zone selection should be automated; otherwise, running the tests will be hard.

Reply:
Yes, the plan still the same, but the idea here is to provide some flexibility, similar to what we did for `PF_API`, if not specified, it will use the prod.
For Port/zone, we gonna follow the same concept.